test/ci-operator-configresolver-integration/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This test runs the ci-operator-configresolver and verifies that it returns
+# properly resolved configs
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOTDIR=$(pwd)
```

Comment:
Use `"$( dirname "${BASH_SOURCE[0]}")"` instead

Reply:
`${BASH_SOURCE[0]}` only gets the relative path, but since we will be changing directory into a tmpdir we need an absolute path.